fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+const url = (path) => `${process.env.REACT_APP_BASE_URL}${path}`;
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    const getJsonData = async () => {
+      try {
+        const response = await fetch(url('/api/todos'));
+        if(!response.ok) throw new Error(`Error Code: ${response.status} Error Message: It Broken`);
+        const dataJson = await response.json();
+        setTodos(dataJson);
+      } catch (err) {
+        setError(err);
+        console.error('Error:', err.message)
```

Comment:
It would be better to log `err` instead of `err.message` because then you would log the entire error, including the stack trace. That would make it easier to debug.